src/Orchard.Web/Modules/Orchard.MediaLibrary/Styles/orchard-medialibrary-admin.css

```diff
@@ -155,6 +155,10 @@
             background-position: center;
         }
 
+        #media-library-main-list.pending li {
+            opacity:.3;
+        }
+

```

Comment:
I wanted a better UI queue to show that the page was loading more content.  Setting the opacity to semi-transparent on the LI items solves this problem as it now lets the background animated gif become more visible during loading of items